packages/mesos-modules/extra/metrics.json

```diff
@@ -0,0 +1,30 @@
+{
+  "libraries": [
+    {
+      "file": "metrics.dll",
```

Comment:
Can you add this configuration to `dcos-config-windows.yaml` like we do in linux version
https://github.com/dcos/dcos/blob/8c6ec915d7653a0fd288148c391454097f3f4bf2/gen/dcos-config.yaml#L362-L366

Reply:
the approach has pending review with @jongiddy ; if we want to be consistent with enterprise, I will change this. call is no Jon side for now.

Reply:
We need to be consistent with Linux and EE version since it will be soon unmaintainable if setups will differ in the way how we configure it. @DominikDary

Reply:
@janisz and @DominikDary there will be a separate pull request to address that issue for linux. at the moment the approach is unknow. this PR is for windows only and doesn't affect anything on linux.

Reply:
I know it's windows only but please be consistent with the rest of the system. What I'm asking is to move this files from `packages/mesos-modules/extra` into `dcos/gen/dcos-config-windows.yaml`. Thats all. So we have at least all modules configured in the same place.

Reply:
right, as pointed in the other comment, that is on @jongiddy to suggest. both ways work right now. if we want to be consistent, we will make this change, no problem.

Reply:
Yes, I think it is reasonable to move this to the `dcos-config-windows.yaml`. The only criteria is that we can install the files in `etc/mesos/modules` without them being deleted by winpanda when it installs the `mesos` module, and I believe that is true now.

Reply:
OK. got it.